ygen/protogen.go

```diff
@@ -337,17 +463,18 @@ func writeProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *g
 // package for the protobuf message(s) that are being generated, such that relative
 // paths can be used in the messages.
 // TODO(robjs): Split the logic of this function into multiple subfunctions.
-func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *genState, cfg protoMsgConfig, parentPkg string) ([]protoMsg, []error) {
+func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *genState, cfg protoMsgConfig, parentPkg string, childMsgs []*generatedProto3Message) ([]protoMsg, []error) {
 	var errs []error
 
 	var msgDefs []protoMsg
 
 	msgDef := protoMsg{
 		// msg.name is already specified to be CamelCase in the form we expect it
 		// to be for the protobuf message name.
-		Name:     msg.name,
-		YANGPath: slicePathToString(msg.path),
-		Enums:    make(map[string]*protoMsgEnum),
+		Name:      msg.name,
+		YANGPath:  slicePathToString(msg.path),
+		Enums:     make(map[string]*protoMsgEnum),
```

Comment:
Keep map init form consistent i.e. either make or {} (btw. the former is preferred per readability comments I've had but I can't find a reference).

Reply:
Not sure where this make came from -- consistently have used the latter throughout the library.